### PR TITLE
feat(CG-1298): add GCP CIS 1.30 2.13

### PR DESCRIPTION
**Pull request by zhouse51**

### CODE REVIEW

File: src/gcp/cis-1.3.0/rules/gcp-cis-1.3.0-2.13.ts [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+`,
+  references: ['https://cloud.google.com/asset-inventory/docs'],
+  gql: `{
+    querygcpAsset {
+      id
+      __typename
+    }
+  }`,
+  resource: 'querygcpProject[*]',
+  severity: 'unknown',
```
Comment (tyler-dunkel) [inline src/gcp/cis-1.3.0/rules/gcp-cis-1.3.0-2.13.ts:57]: @m-pizarro is severity unknown a valid value here? How would the UI treat this?

Comment (zhouse51) [reply src/gcp/cis-1.3.0/rules/gcp-cis-1.3.0-2.13.ts:60]: @tyler-dunkel There are several new added rules has the unknown severity. Maybe we should review them together.

Comment (m-pizarro) [reply src/gcp/cis-1.3.0/rules/gcp-cis-1.3.0-2.13.ts:60]: @tyler-dunkel @james-zhou-inspire11  Yes, initially for the rules that didn't have the IG set, we set the `unknown` value, but later we decided to set it as `medium` for those cases, and as you say, there may be rules that have not been updated. 

This is the severity mapping:
- IG1 => `high`
- IG2 => `medium`
- IG3 => `low`
- No IG set (unknown) => `medium`

Comment (tyler-dunkel) [reply src/gcp/cis-1.3.0/rules/gcp-cis-1.3.0-2.13.ts:60]: ok sounds good

**Pull request merged**